tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -198,6 +198,31 @@ _ when commandTypeName.EndsWith(commandSuffix) =>
             }
         }
 
+        internal static bool IsDisabledCommandType(string commandTypeName)
```

Comment:
Can we instead pass the `Tracer` object through the method parameters so this method doesn't rely on the static `Tracer.Instance`? WDYT?

Reply:
Sounds good to me!
Needed to propagate the parameter up a bit as well.

Reply:
This has proven to be a bit of a headache as I have ultimately needed to pass `Tracer.Instance` to the static `Cache` class

Without doing more refactoring I may just revert this and depend on the static `Tracer.Instance` for now and come back later to refactor.

Reply:
Yeah @zacharycmontoya I've reverted this, I'll look at it later (follow up PR) if that sounds okay?